library/idle.po

```diff
@@ -515,16 +515,15 @@ msgid "Strip trailing whitespace"
 msgstr "*Strip trailing whitespace*"
 
 #: ../Doc/library/idle.rst:200
-#, fuzzy
 msgid ""
 "Remove trailing space and other whitespace characters after the last non-"
 "whitespace character of a line by applying str.rstrip to each line, "
 "including lines within multiline strings.  Except for Shell windows, remove "
 "extra newlines at the end of the file."
 msgstr ""
-"Enlève les espaces et autres caractères blancs après le dernier caractère "
-"non blanc d'une ligne en appliquant *str.rstrip* à chaque ligne, y compris "
-"les lignes avec des chaînes de caractère multi-lignes."
+"Enlève les espaces et autres caractères d'espacement après le dernier "
```

Comment:
`caractères blancs` n'était peut être pas la meilleure traductions, mais `les espaces et autres caractères d'espacement` ne me plait pas plus, trop redondant. Tu peux cependant largement simplifier, avec quelque chose comme :

    Enlève les espaces après le dernier caractère non blanc ...

Reply:
Tu as oublié de traduire la phrase :

    Except for Shell windows, remove extra newlines at the end of the file.

Reply:
Ok